src/app/application.tsx

```diff
@@ -247,8 +254,18 @@ export class Application {
       this.appStore.dispatcher.dispatch(new Actions.ImportDataset(dataset));
     } else if (hashParsed.loadCSV) {
       // Quick load from one or two CSV files
+      // default to comma delimiter, and en-US number format
+      const localeFileFormat: LocaleFileFormat = {
+        delimiter: ",",
+        numberFormat: {
+          remove: ",",
+          decimal: "."
+        }
+      };
       const spec: DatasetSourceSpecification = {
-        tables: hashParsed.loadCSV.split("|").map(x => ({ url: x }))
+        tables: hashParsed.loadCSV
```

Comment:
I know you didn't do this, but double checking `hashParsed.loadCSV` isn't a function right? The name implies an action.  I assume from it's usage that it is the loaded csv.

Reply:
Yeah I also wasn't sure what this was. After looking, it's a feature that loads via url. So hashParsed is a dictionary of string values, and .loadCSV is a string.